### PR TITLE
prepare mros2-posix for evaluation

### PR DESCRIPTION
**Pull request by Haru-T**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -10,3 +10,6 @@
 	path = micro-ros-stm32_embeddedrtps/micro-ros_embeddedrtps/micro_ros_stm32cubemx_utils
 	url = https://github.com/mROS-base/micro_ros_stm32cubemx_utils
 	branch = galactic
+[submodule "mros2-posix"]
+	path = mros2-posix
+	url = git@github.com:mROS-base/mros2-posix.git
```
Comment (takasehideki) [inline .gitmodules]: It's better to denote `branch` or `tag` to clarify the version for evaluation. Please specify it.

Comment (Haru-T) [reply .gitmodules]: Done: https://github.com/mROS-base/eval/pull/3/commits/25949ca05f259dd90840e8f82fa978d152fc79b4

File: .gitmodules [hunk 2]
```diff
@@ -10,3 +10,7 @@
 	path = micro-ros-stm32_embeddedrtps/micro-ros_embeddedrtps/micro_ros_stm32cubemx_utils
 	url = https://github.com/mROS-base/micro_ros_stm32cubemx_utils
 	branch = galactic
+[submodule "mros2-posix"]
+	path = mros2-posix
+	url = git@github.com:mROS-base/mros2-posix
+	branch = eval/v0.1.0
```
Comment (takasehideki) [inline .gitmodules:16]: README and directory seem to be `eval/v0.2.0` (differ to this). My suggestion is to create `eval/v0.2.0` branch on mros2-posix repository for consistency.

File: .gitmodules [hunk 3]
```diff
@@ -10,3 +10,7 @@
 	path = micro-ros-stm32_embeddedrtps/micro-ros_embeddedrtps/micro_ros_stm32cubemx_utils
 	url = https://github.com/mROS-base/micro_ros_stm32cubemx_utils
 	branch = galactic
+[submodule "mros2-posix"]
+	path = mros2-posix
+	url = git@github.com:mROS-base/mros2-posix
```
Comment (takasehideki) [inline .gitmodules]:
```suggestion
	url = https://github.com/mROS-base/mros2-posix
```

Of course I understand ssh is better than https. But for those unfamiliar with GitHub, this notation can cause trouble (in fact, I have dealt with people who have trouble with this several times :D

File: .gitmodules (hunk 2 above)
Comment (Haru-T) [reply .gitmodules:16]: I assumed that `v0.2.0` in README.md and the directory name corresponds to the version of `eval` repo (and the merged commit will be tagged `v0.2.0`). Does it make sense?

Comment (takasehideki) [reply .gitmodules:16]: Oops,  you're right! sorry for my lack of memory,,,

**Review by takasehideki — APPROVED** [review]

**Comment by takasehideki** [comment]
@Haru-T ❤️❤️❤️❤️

**Pull request merged**